share/multiline.py

```diff
@@ -85,7 +88,14 @@ def collect_and_reset(self) -> tuple[bytes, int]:
 
         self.previous = b""
 
-        return data, current_length
+        if data.find(b"\r\n") > -1:
```

Comment:
what's the reason for this addition? i think a comment here would help future readers.

Reply:
answered [here](https://github.com/elastic/elastic-serverless-forwarder/pull/534/files#diff-2e204b34a51f1f25840268e768056fd46938d1ae5320dcf0866c6092d8614c10R23-R25)